spiffe/build.rs

```diff
@@ -1,22 +1,29 @@
-use std::{env, fs};
+use std::env;
 
-fn main() -> Result<(), anyhow::Error> {
-    // Check if this is a docs.rs build
-    let is_docs_rs = env::var_os("DOCS_RS").is_some();
+use anyhow::Context as _;
 
-    if !is_docs_rs {
-        let mut proto_config = prost_build::Config::new();
-        proto_config.bytes(["."]);
-        let file_descriptors = protox::compile(["src/proto/workload.proto"], ["src/proto"])?;
-        tonic_build::configure()
-            .build_client(true)
-            .out_dir("src/proto")
-            .compile_fds_with_config(proto_config, file_descriptors)?;
+fn main() -> anyhow::Result<()> {
+    println!("cargo::rerun-if-changed=src/proto");
+    println!("cargo::rerun-if-env-changed=DOCS_RS");
 
-        fs::rename("src/proto/_.rs", "src/proto/workload.rs")?;
-    } else {
+    // Check if this is a docs.rs build
+    if env::var_os("DOCS_RS").is_some() {
         println!("cargo:warning=Skipping protobuf code generation on docs.rs.");
+        return Ok(());
     }
+    let out_dir = env::var_os("OUT_DIR").context("failed to lookup `OUT_DIR`")?;
```

Comment:
Initialize `out_dir` for use in the file renaming step below:

```suggestion
    let out_dir = env::var_os("OUT_DIR").context("failed to lookup `OUT_DIR`")?;
    let out_dir = Path::new(&out_dir);
```